src/e2ee.rs

```diff
@@ -659,113 +663,120 @@ unsafe fn decrypt_part(
     public_keyring_for_validate: &Keyring,
     ret_valid_signatures: &mut HashSet<String>,
     ret_decrypted_mime: *mut *mut Mailmime,
-) -> bool {
-    let mut ok_to_continue = true;
+) -> Result<bool> {
```

Comment:
I'd appreciate if you could add a docstring about what this function does and especially what it's return value means.  From this I'm completely at a loss what `Ok(true)` or `Ok(false)` would signify.

Reply:
i haven't written that function and am not totally sure what it does exavctly, sorry. 
I have a rough idea but if i add a docstring but get things wrong readers will just be confused. 
So if i can i'll see to add docstrings but the whole incoming-message chain is not clear to me, and which parts do what.

Reply:
There's nothing wrong with a partial docstring btw, just saying what the return value does would be helpful already, and that is what you're changing.